dv-clone.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2024 The Kube-burner Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ocp
+
+import (
+	"os"
+
+	"github.com/kube-burner/kube-burner/pkg/workloads"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	dvCloneTestName                = "dv-clone"
+	dvCloneDefaultContainerDiskUrl = "quay.io/yblum/tiny_image:latest"
```

Comment:
Let me give you permissions on kube-burner's quay.io project

Reply:
That's even better

Reply:
I uploaded the image to the kube-burner org and updated the test